packages/grafana-ui/src/components/Graph/GraphWithLegend.story.tsx

```diff
@@ -79,64 +99,58 @@ const series: GraphSeriesXY[] = [
   },
 ];
 
-const getStoriesKnobs = () => {
-  const rightAxisSeries = text('Right y-axis series, i.e. A,C', '');
-
-  const legendPlacement = select<LegendPlacement>(
-    'Legend placement',
-    {
-      bottom: 'under',
-      right: 'right',
-    },
-    'bottom'
-  );
-  const renderLegendAsTable = select<any>(
-    'Render legend as',
-    {
-      list: false,
-      table: true,
-    },
-    false
-  );
-
-  return {
-    rightAxisSeries,
-    legendPlacement,
-    renderLegendAsTable,
-  };
-};
+interface StoryProps extends GraphWithLegendProps {
+  rightAxisSeries: boolean;
+  displayMode: string;
+}
 
-export const graphWithLegend = () => {
-  const { legendPlacement, rightAxisSeries, renderLegendAsTable } = getStoriesKnobs();
-  const props: GraphWithLegendProps = {
+export const WithLegend: Story<StoryProps> = ({ rightAxisSeries, displayMode, legendDisplayMode, ...args }) => {
+  const props: Partial<GraphWithLegendProps> = {
     series: series.map((s) => {
       if (
         rightAxisSeries
-          .split(',')
-          .map((s) => s.trim())
-          .indexOf(s.label.split('-')[0]) > -1
+          ? 'A,C'
+          : ''
+              .split(',')
+              .map((s) => s.trim())
+              .indexOf(s.label.split('-')[0]) > -1
```

Comment:
Not sure we need this code any longer, I think you need to choose between:
- either we keep the ability to enter which series that should have a right axis as before see [https://developers.grafana.com/ui/latest/index.html?path=/story/visualizations-graph-graphwithlegend--graph-with-legend](https://developers.grafana.com/ui/latest/index.html?path=/story/visualizations-graph-graphwithlegend--graph-with-legend)
![image](https://user-images.githubusercontent.com/562238/112311912-7f1d0480-8ca6-11eb-9440-bf8995091762.png)

- or keep it like this and remove "the split code" above

Reply:
Yeah...I get your point @hugohaggmark 
Let's keep it as it is before.

Would push some updates right away.
Thanks! 👍